v2/yarpcthrift/inbound_test.go

```diff
@@ -120,43 +124,43 @@ func TestDecodeRequestEncodingError(t *testing.T) {
 	defer cancel()
 
 	// XXX handler and protocol superfluous for this case
-	h := thriftUnaryHandler{}
-	rw := new(transporttest.FakeResponseWriter)
+	h := UnaryHandler{}
 	req := request()
 	// XXX bogus encoding override
-	req.Encoding = transport.Encoding("bogus")
-	err := h.Handle(ctx, req, rw)
+	req.Encoding = yarpc.Encoding("bogus")
+	res, resBody, err := h.Handle(ctx, req, requestBody())
 
+	assert.Nil(t, res)
+	assert.Nil(t, resBody)
 	if assert.Error(t, err, "expected an error") {
```

Comment:
I know that this was from before but we could change this to a `require` and drop the `if`